src/documentation/pages/components/forms/search-input/examples/_sizes-lumen-centurylink.vue

```diff
@@ -0,0 +1,158 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes-lumen-centurylink" :tabs="exampleTabs">
+    div(slot="example-description")
+      p.-text Search input supports the following sizes: <code>xs</code>, <code>sm</code>, <code>md</code>,
+        |  <code>lg</code>, <code>xl</code>.
+        | The default size is <code>md</code>.
+    .-p--0(slot="example")
+      .chi-grid
+      each size in ['xs', 'sm', 'md', 'lg', 'xl']
```

Comment:
We should use v-for loops here, something like that. 
```suggestion
      .chi-grid(v-for="size in ['xs', 'sm', 'md', 'lg', 'xl']")
```
You can take a look in other examples.

Reply:
This will not be needed in this file and below files as there are chi components utilized within it. I have referred other files and we are not using it there as well.

Reply:
I did the v-for changes and updated the PR.